### PR TITLE
Update Algorithm Find The Longest Word In A String

### PR DESCRIPTION
**Pull request by koustuvsinha**
Refer #1094

### CODE REVIEW

File: Algorithm-Find-The-Longest-Word-In-A-String.md [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
+## :speech_balloon: Hint: 2
 
 You will need to figure out a way to keep track globally of the greatest current length.
 
-## Hint: 3
+> _try to solve the problem now_
+
+## :speech_balloon: Hint: 3
 
 Remember how to get the length of elements on the array? `Array[index].length`
```
Comment (atjonathan) [inline Algorithm-Find-The-Longest-Word-In-A-String.md]: Add a full stop at the end of this sentence.

File: Algorithm-Find-The-Longest-Word-In-A-String.md [hunk 2]
```diff
@@ -93,25 +125,24 @@ function findLongestWord(str) {
...
-For more information on `reduce` [click here.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)<br>
-In case you're wondering about that `0` after the callback function, it is used to give an initial value to the `x`, so that `Math.max` will know where to start.
+- [JS Functions](JS-Functions.md)
+- [Recursion Basics](https://www.youtube.com/watch?v=k7-N8R0-KY4)
 
-## Third solution
+### :trophy: Credits:
 
-See the code's inline comments.
+If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`Thanks @Rafase282 @shadowfool @Hallaathrad for your help with Algorithm: Find the Longest Word in a String`**
```
Comment (atjonathan) [inline Algorithm-Find-The-Longest-Word-In-A-String.md]: Add two lines between `on the main chat:` and `Thanks`:
```diff
- If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`Thanks @Rafase282 @shadowfool @Hallaathrad for your help with Algorithm: Find the Longest Word in a String`**
+ If you found this page useful, you can give thanks by copying and pasting this on the main chat:
+
+ **`Thanks @Rafase282 @shadowfool @Hallaathrad for your help with Algorithm: Find the Longest Word in a String`**
 
```

**Comment by Rafase282** [comment]
Add the code explanations, feel free to use the in-code comments as a guide.

**Comment by Rafase282** [comment]
https://github.com/koustuvsinha/wiki/pull/3

**Comment by Rafase282** [comment]
# LGTM :+1:

**Pull request merged**